packages/visualizations/src/components/Map/utils.js

```diff
@@ -43,7 +43,13 @@ export const colorShapes = (geoJson, values, colorsScale) => {
     const coloredFeatures = geoJson.features.map((feature) => {
         const shapeMapping = feature.properties.key;
         const value = dataMapping[shapeMapping]; // FIXME: beware of int/string differences in keys
-        const color = scale(value).hex();
+        let color;
+        const applyDifferentEmptyValueColor = !value && defaultEmptyValueColor;
+        if (applyDifferentEmptyValueColor) {
+            color = chroma(defaultEmptyValueColor).hex();
+        } else {
+            color = scale(value).hex();
+        }
```

Comment:
I like this change, it makes the default applied color more explicit rather than using the one from chroma.js ! 👍  I'll probably keep the same as from chroma, id est : #cccccc to avoid regression !